notebooks/src/el-paso.ipynb

```diff
@@ -272,6 +272,26 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "def get_historical_data(date: date, column_name):\n",
+    "    \"\"\"\n",
+    "    Look up the value of a parameter on a given date\n",
+    "    in the historical data we've loaded.\n",
+    "    \n",
+    "    Return the value or raise a KeyError if we don't have it.\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    # prefer Texas government data over @KrisMoore compiled data\n",
+    "    try: \n",
+    "        value = el_paso_cases.loc[date, column_name]\n",
+    "        if np.isnan(value):\n",
+    "            raise KeyError\n",
+    "        return value\n",
```

Comment:
A single `return value` at the end of the function would also work.

Reply:
👍